webapp/src/common/types.ts

```diff
@@ -563,11 +563,21 @@ export interface UpdateAreaUi {
   color_rgb: Array<number>;
 }
 
-export interface LinkCreationInfo {
+export interface LinkCreationUIDTO {
```

Comment:
rename LinkUIInfoDTO

Reply:
`LinkInfoDTO` instead we have `LinkInfoDTOWithUI`